src/ice_shelf/MOM_ice_shelf.F90

```diff
@@ -1781,7 +2024,7 @@ subroutine ice_shelf_save_restart(CS, Time, directory, time_stamped, filename_su
   if (present(directory)) then ; restart_dir = directory
   else ; restart_dir = CS%restart_output_dir ; endif
 
-  call save_restart(restart_dir, Time, CS%grid, CS%restart_CSp, time_stamped)
+  call save_restart(restart_dir, Time, CS%grid_in, CS%restart_CSp, time_stamped)
```

Comment:
A generalized `save_restart` for rotation and nonrotation was never implemented (unlike `time_interp_external`) but this is revealing its absence.

I don't think we need to implement this for the PR, but might be worth adding a TODO comment mentioning that we need a `rotated_save_restart` (or whatever).

Reply:
Can we put these changes in a separate PR?

Reply:
Yes that's fine.